script.js

```diff
@@ -1,16 +1,27 @@
 // Write your JavaScript code here!
 
-window.addEventListener("load", function() {
+window.addEventListener("load", function () {
+    let list = document.querySelector("div[id=faultyItems]");
```

Comment:
```suggestion
    let list = document.getElementById("faultyItems");
```
For getting the list, you can use `document.getElementById()` instead.